haiku-apps/spybmessage/spybmessage-0.0~git.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A command line tool to monitor BMessage events received by an application"
+DESCRIPTION="The spybmessage command is primarily useful as a debugging tool when writing graphical
+Haiku applications.  When run from a Terminal it will print out any BMessage events that the target
+application receives.
+
+This command comes from the Haiku source code repository but it is not included in normal builds of
+Haiku."
+HOMEPAGE="https://www.haiku-os.org/"
+COPYRIGHT="2008 François Revol (mmuman)"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="01f3ea96af191482839e6389621d45e56782618c"
+SOURCE_URI="https://raw.githubusercontent.com/haiku/haiku/$srcGitRev/src/bin/spybmessage.cpp#noarchive"
```

Comment:
Wouldn't it be more appropriate to package this in e.g. haiku_extras than in haikuports? It is kind of weird to have something from the Haiku core repository in haikuports.

Reply:
Yes, it makes more sense to keep stuff from Haiku git repository packaged by Haiku.

Or otherwise, move it out of Haiku repositories.

Reply:
I got no response from the merge request at gerrit to have this included.  Whether it was in the main image or in some extra package.  I wanted it to be available.

Reply:
That change request at haiku was abandoned before I even had a chance to see it (admittedly I've not been very active there lately).